style.css

```diff
@@ -1,24 +1,23 @@
 .main-page {
   height: 718px;
   width: 375px;
```

Comment:
- [ ] Please avoid giving static width for the website. The website should fill the entire width and rearrange itself on breakpoints. I think the extra space in the picture should be removed. 
![picture](https://user-images.githubusercontent.com/12524453/177865785-3e1d2a6e-0b1a-4bf3-9407-0325961ae82f.png)